src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -305,7 +307,7 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', function (c3Cha
 
           // default to 'used' info.
           centerLabelText = { bigText: $scope.data.used,
-                              smText:  $scope.config.units + ' Used' };
+                              smText:  $i18next('pfDonutPctChart.used', { units: $scope.config.units }) };
```

Comment:
Hmm, I suppose it's a trade off.  As the user I'd have to specify 'available' and 'used' several times if I used more than one type of chart (for example).  And it also makes it more difficult for a developer to follow where text comes from as there is an extra level of abstraction.

I wonder if we could add the context in the string extraction process.  Gettext is nice for this sort of thing because it tells you the filename (and line) that the string comes from so it provides context that way.